contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -714,6 +714,128 @@ $BODY$
 STRICT
 LANGUAGE SQL IMMUTABLE;
 
+CREATE OR REPLACE FUNCTION sys.DATETIMEOFFSETFROMPARTS(IN p_year NUMERIC,
```

Comment:
Is there a reason to use `NUMERIC`? The official document says just `integer expression` (https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/t-sql/functions/datetimeoffsetfromparts-transact-sql.md).